scripts/lib/install

```diff
@@ -195,13 +210,35 @@ fi
 # don't run dist-upgrade in one click apps to make the
 # installation process more seamless.
 if [ -z "$NO_DIST_UPGRADE" ]; then
-    apt-get -y dist-upgrade "${APT_OPTIONS[@]}"
+    case $os_distro in
+        debian)
+            apt-get -y dist-upgrade "${APT_OPTIONS[@]}"
+            ;;
+        *)
+            echo "OS major upgraded not supported on $os_id. Skippin"
```

Comment:
I think this should just be a `yum upgrade`; the intent is to not be running old packages.  `dist-upgrade` doesn't do major upgrades; it's just a more thorough `apt upgrade`.